samples/word/99-preview-apis/manage-fields.yaml

```diff
@@ -0,0 +1,151 @@
+id: word-manage-fields
+name: Get fields
+description: This sample shows how to get existing fields.
+host: WORD
+api_set:
+    WordApi: '1.4'
+script:
+    content: |
+        $("#get-first").click(() => tryCatch(getFirstField));
```

Comment:
For purposes of Script Lab snippet imports, consider adding a one-sentence description to each function under the `Word.run`.